packages/payment-processor/test/payment/erc777-stream.test.ts

```diff
@@ -92,31 +92,6 @@ describe('erc777-stream', () => {
     });
   });
 
-  describe('Superfluid framework', () => {
-    it.each([
-      { network: 'goerli' },
-      { network: 'matic' },
-      // { network: 'xdai' },
-      { network: 'optimism' },
-      { network: 'avalanche' },
-      { network: 'arbitrum-one' },
-    ] as Array<{ network: CurrencyTypes.EvmChainName }>)(
-      'Should initialize superfluid framework on $network',
-      async ({ network }) => {
-        const provider = getDefaultProvider(network);
-        const networkValidRequest = {
-          ...validRequest,
-          currencyInfo: {
-            ...validRequest.currencyInfo,
-            network,
-          },
-        };
-        const sf = await getSuperFluidFramework(networkValidRequest, provider);
-        expect(sf).toBeDefined();
-      },
-    );
-  });
```

Comment:
This is definitely not a "unit" test. Considering the flakiness it creates, I'm dropping it and not replacing it.